integration/apps/rails-six/config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
+
+# ActiveSupport::Reloader.to_prepare do
+#   ApplicationController.renderer.defaults.merge!(
+#     http_host: 'example.org',
+#     https: false
+#   )
+# end
```

Comment:
Minor: (Similar suggestion as for rails 5 -- since we're probably never going to use any of these commented-out examples under `initializers/`, perhaps we could just remove them?)